_rules/text-spacing-not-important-b8bb68.md

```diff
@@ -22,33 +22,33 @@ acknowledgments:
 
 This rule applies to any HTML element that is [visible][] and has a [declared][] value specified via the `style` attribute for any of the following CSS properties:
 
-- [word-spacing][],
-- [letter-spacing][],
+- [word-spacing][]; or
+- [letter-spacing][]; or
 - [line-height][].
```

Comment:
```suggestion
- the `style` attribute declares [word-spacing][]; or
- the `style` attribute declares [letter-spacing][]; or
- the `style` attribute declares [line-height][].
```

Reply:
I took your lead & updated the applicability:

```
This rule applies to any HTML element that is [visible][], for which one of the following is true:

- the `style` attribute [declares][declared][] [word-spacing][] CSS property; or
- the `style` attribute [declares][declared][] [letter-spacing][] CSS property; or
- the `style` attribute [declares][declared][] [line-height][] CSS property.
```